fuel-tx/src/transaction/consensus_parameters/gas.rs

```diff
@@ -949,6 +1057,25 @@ impl GasCostsValues {
             GasCostsValues::V1(v1) => v1.swwq,
             GasCostsValues::V2(v2) => v2.swwq,
             GasCostsValues::V3(v3) => v3.swwq,
+            GasCostsValues::V4(v4) => v4.swwq,
+        }
+    }
+
+    pub fn bsiz(&self) -> DependentCost {
+        match self {
+            GasCostsValues::V1(_v1) => todo!(), // TODO
+            GasCostsValues::V2(_v2) => todo!(), // TODO
+            GasCostsValues::V3(_v3) => todo!(), // TODO
+            GasCostsValues::V4(v4) => v4.bsiz,
+        }
+    }
+
+    pub fn bldd(&self) -> DependentCost {
+        match self {
+            GasCostsValues::V1(_v1) => todo!(), // TODO
+            GasCostsValues::V2(_v2) => todo!(), // TODO
+            GasCostsValues::V3(_v3) => todo!(), // TODO
```

Comment:
I'm not sure what should be done for gas prices on old versions, where this opcode doesn't exist yet. Possible options include:

- `0`, carries risk of free gas use and generally seems not suitable
- `u64::MAX`, causes gas to immediately run out which would at least disallow transacting with incorrect behavior
- rust-level panic, and to disallow incorrect configurations somewhere else
- vm-level panic

Reply:
Changed to VM level panic.